docs/source/distributed.rst

```diff
@@ -7,7 +7,7 @@ Helper module to use distributed settings for multiple backends:
 
 - XLA on TPUs via `pytorch/xla <https://github.com/pytorch/xla>`_
 
-- using `Horovod distributed framework <https://horovod.readthedocs.io/en/stable/>`_ as a backend
+- using `Horovod framework <https://horovod.readthedocs.io/en/stable/>`_ as a backend
```

Comment:
This has changed to `Horovod framework` as there is `Horovod distributed framework` in `launcher.py`

Reply:
Sounds good !